### PR TITLE
Updates to Blue Waters guide on website

### PR DESCRIPTION
**Pull request by LukeSeifert**
## Summary of changes
There are a few minor changes to the Blue Waters guide on the arfc website. These include fixing the location of the group, links, and an extra added tip about running jobs.

## Types of changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] I have read the **CONTRIBUTING** document.
- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

## Associated Issues and PRs

- Issue: #216 

## Associated Developers

- Dev: @katyhuff 

## Checklist for Reviewers

Reviewers should use [this link](http://arfc.github.io/manual/guides/pull_requests) to get to the 
Review Checklist before they begin their review.

### CODE REVIEW

File: manual/guides/blue_waters.md [hunk 1]
```diff
@@ -30,24 +31,19 @@ follow the instruction sent to you by email.
...
 ## Step 4. Activating Token
-With your token, open the email 'Blue Waters Instructions'
-Click the [link](https://otp.ncsa.illinois.edu)
-and follow its directions, and activate your token.
+For activating the token, instructions should be contained within the emails recieved. Extra information can be found [here](https://wiki.ncsa.illinois.edu/display/cybersec/Duo+at+NCSA).
 
-** tip) the pin must be 8 digits, letters and numbers, something that you'll never forget!
 
 
 ## Step 5. Loggin In
```
Comment (katyhuff) [inline manual/guides/blue_waters.md:38]:
```suggestion
## Step 5. Logging In
```

File: manual/guides/blue_waters.md [hunk 2]
```diff
@@ -30,24 +31,19 @@ follow the instruction sent to you by email.
...
 Open terminal, type
 ```
 $ssh [username]@h2ologin.ncsa.illinois.edu
 ```
+You will then be prompted for your username, password, token selection, and then your token input.
```
Comment (katyhuff) [inline manual/guides/blue_waters.md:43]: I'm confused by this sentence. When SSH-ing in, you've already given your username (in the ssh command above) and there's no need to select a token (that was done in steps 2,3, & 4.) At this point, you just type in your pin along with whatever number appears on your token. What am I missing?

**Review by katyhuff — CHANGES_REQUESTED** [review]

File: manual/guides/blue_waters.md (hunk 2 above)
Comment (LukeSeifert) [reply manual/guides/blue_waters.md:43]: Ah, the username part was a mistake. When I SSH in though, it has a list (though there is only one option) where it says "Enter a passcode or select one of the following options" followed by "1. Duo Push to XXX-XXX-0603". After I type "1" to select that option, I can accept the request from Duo on my phone and gain access. I think this is some sort of token selection list, where if someone has a a physical token and Duo they could use either.

Comment (katyhuff) [reply manual/guides/blue_waters.md:43]: ah, I see. I forgot about that. Maybe clarify all that here.

File: manual/guides/blue_waters.md [hunk 3]
```diff
@@ -35,12 +35,12 @@ For activating the token, instructions should be contained within the emails rec
...
 
 
-## Step 5. Loggin In
+## Step 5. Logging In
 Open terminal, type
 ```
 $ssh [username]@h2ologin.ncsa.illinois.edu
 ```
-You will then be prompted for your username, password, token selection, and then your token input.
+You will then be prompted for your password, followed by an option to select from a list of your registered tokens. After selecting the token you will use (if you only have one token, simply type 1), either enter your pin and token number or approve the login request with Duo, depending on what token type you are using.
```
Comment (katyhuff) [inline manual/guides/blue_waters.md:43]: limit line lengths.
```suggestion
You will then be prompted for your password, followed by an option to select from a list of your 
registered tokens. After selecting the token you will use (if you only have one token, simply 
type 1), either enter your pin and token number or approve the login request with Duo, 
depending on what token type you are using.
```

**Review by katyhuff — CHANGES_REQUESTED** [review]

**Review by katyhuff — APPROVED** [review]

**Comment by katyhuff** [comment]
Nice work!!

**Pull request merged**